BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -1,4 +1,5 @@
 import BlueprintUI
```

Comment:
General question / thought: if you haven't already, it would be worth running these changes through instruments, plus ensuring that when there aren't links, we're not going through newer expensive code paths for handling links? Given this is the base of every label in the app, ensuring we're not adding additional work for the 99% case of no links will be super important

Reply:
- [ ] Profile